git-server/src/lib.rs

```diff
@@ -112,6 +113,10 @@ impl Context {
 pub async fn run(options: Options) {
     let ctx = Context::try_from(&options).expect("Failed to create context from service options");
 
+    let mut aliases = HashMap::new();
+    grab_aliases(&options, &mut aliases);
+    let aliases = Arc::new(aliases);
```

Comment:
Aliases should go inside `Context`, then `grab_aliases` can be a method on `Context`.

Reply:
We should also use `Urn` as the value to make use of the type checker.

Reply:
Fixed https://github.com/radicle-dev/radicle-client-services/pull/74/commits/8e795c8bba5f1e1edb21d3dd531c6a8bea3d75ff#diff-b672a15ad14f54528cebd77aee76d3007962d44bda987b3e0a34597ec80fd6b1R48